cluster/addons/cluster-monitoring/google/heapster-controller.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: "v1beta3"
```

Comment:
You don't need to quote any of the strings in the yaml files (except perhaps if the string is "true", judging by the deleted files).